e2e_playwright/st_plotly_chart_select_test.py

```diff
@@ -95,24 +95,31 @@ def test_box_select_on_stacked_bar_chart_displays_a_df(app: Page):
     expect(app.get_by_test_id("stDataFrame")).to_have_count(1)
 
 
-# TODO(willhuang1997): Readd choropleth charts and add a working test
 # This test could be flakey because https://github.com/plotly/plotly.js/issues/6898
-# def test_click_on_choroleth_chart_displays_a_df(app: Page):
-#     chart = app.locator(".stPlotlyChart").nth(4)
-#     chart.scroll_into_view_if_needed()
-#     expect(chart).to_be_visible()
-#     chart.hover()
-#     app.mouse.down()
-#     app.mouse.up()
-#     expect(app.get_by_test_id("stDataFrame")).to_have_count(1)
+# Mapbox doesn't load on firefox so just test on chrome for now
+@pytest.mark.only_browser("chromium")
+def test_box_select_on_choroleth_chart_displays_a_df(app: Page):
+    chart = app.locator(".stPlotlyChart").nth(4)
+    chart.scroll_into_view_if_needed()
+    expect(chart).to_be_visible()
+
+    # wait for map to load
+    wait_for_app_run(app, 3000)
+
+    chart.hover()
+    app.mouse.down()
+    app.mouse.move(50, 50)
+    app.mouse.move(150, 150)
+    app.mouse.up()
```

Comment:
I believe we need a `wait_for_app_run` here since the up is triggering a rerun or?